core/src/CopyButton/CopyButton.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from "react";
+import cx from "classnames";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faClipboard, faCheck } from "@fortawesome/free-solid-svg-icons";
+
+import { Button } from "../Button";
+
+import styles from "./CopyButton.less";
+
+export const CopyButton = (props) => {
+  const [copied, setCopied] = useState(false);
+
+  function fallback(text) {
+    if (window.clipboardData && window.clipboardData.setData) {
+      // IE specific code path to prevent textarea being shown while dialog is visible.
+      return window.clipboardData.setData("Text", text);
+    } else if (
+      document.queryCommandSupported &&
+      document.queryCommandSupported("copy")
+    ) {
+      const textarea = document.createElement("textarea");
+      textarea.innerText = text;
+      document.body.appendChild(textarea);
```

Comment:
Why are we appending this textarea to the document?

Reply:
@shrunyan In order for the method .select() to work properly the input/textarea needs to be appended into the dom. This is from reading online and referencing your example FieldTypeUUID

Reply:
Does this cause any display issues? I imagine the textarea would assume the space of it's content causing layout issues.

Reply:
No display issue as far I tested.

Reply:
Did you test in a browser which does not support `navigator.clipboard`?

Reply:
Yes and fall back is working properly as well.